.eslintrc.yml

```diff
@@ -0,0 +1,20 @@
+env:
+  browser: true
+  es6: true
+  node: true
+extends: 'eslint:recommended'
+parserOptions:
+  ecmaVersion: 2018
```

Comment:
which ES version fits best the node 8.4, which is defined in package.json?
8 Node LTS quits on https://node.green/#ES2017
2018 fits for 10 LTS
Not sure which fits best to be synchrony with serialport:
* A: 8 LTS @ ES2016
* B: 10 LTS @ ES2018

Reply:
I used 2018 because of the spread operator i use 
[here](https://github.com/RafaelKa/node-serialport-enocean-parser/blob/74d584596af106d7e2ecaabd5fc73262d0ba9783/ESP3Parser.js#L21)
this is part of 2018 and runs with 8.4. but we can go lower if you like. 
Node-Red (our only real current users) recommend node 8.x LTS so we are good there as well.
It's not about supporting all of 2018, but to validate the features we use from it that run with our min-version. We test if our code runs with 8.4 anyways... so...